### DISCUSSION TITLE
Search window appears for single trigger

### DISCUSSION THREAD

**Discussion by lduperval** [espanso/espanso#1995] [Q&A]
Hi,

I don't understand why the search window shows up for a trigger that exists only once in my configuration file. I added a label to see what was happening. The Alt-1 choice shows the trigger, the Alt-2 choice shows the label I added.

Where is Alt-1 coming from? There is only one trigger (so far) that does this.

L

**Comment by smeech** [comment]
Did you resolve this?

If not, can you post your trigger code and an image of the search window?

**Reply by lduperval** [reply]
No, the problems is still there:

- trigger: .dupcg.
  replace: myemail@gmail.com
  label: the only one

![2024-08-11 13_20_52-Clipboard](https://github.com/user-attachments/assets/d7872d66-6044-4304-bca1-7520abe41479)

I added the label to see if it was picking up the information elsewhere but it isn't. 

L

**Reply by lduperval** [reply]
Is there a way to increase the logging level so I can see what espanso is trying to do which makes it show the window?

**Comment by smeech** [comment]
Can we double-check you don't have a duplicate trigger (the usual cause of the search window popping up) by typing:
```
espanso match list | grep dupcg
```
in Linux, or the PowerShell command:
```
espanso match list | Select-String "dupcg"
```
if you're using Windows?

Does the problem go away if you change the trigger?

**Comment by lduperval** [comment]
No, I don't... Dang! There was one in base.yml. 🤦🏽‍♂️🤦🏽‍♂️🤦🏽‍♂️

Wow, why didn't I think of that? I suspect I might have put one there when I first installed espanso and didn't know what I was doing. Then I promptly forgot about it and only worked in the file I had created.

Wow…

Thanks!

L

**Reply by smeech** [reply]
Excellent! Glad it was a simple problem.

As you use Espanso you'll find it useful to keep different `.yml` files for different purposes, and `base.yml` is one I don't touch these days.
